apps/api/src/app/user/update-user-setting.dto.ts

```diff
@@ -48,6 +49,14 @@ export class UpdateUserSettingDto {
   @IsOptional()
   locale?: string;
 
+  @IsNumber()
+  @IsOptional()
+  projectedTotalAmount?: number;
+
+  @IsDate()
```

Comment:
```suggestion
  @Type(() => Date)
  @IsDate()
```

Reply:
Thanks a lot for the hint. I was not able to solve it with `@Type(() => Date)` though. But it's working now the same way as `date` in `CreateOrderDto`.